### DISCUSSION TITLE
How do you write a prisma query in which the filter is dependent on the sum of two columns?

### DISCUSSION THREAD

**Discussion by francis-chang** [prisma/prisma#16155] [Q&A] [answered]
I have a Raw SQL query like so

`select  * from foo where  bar  + baz> 0;`

how would you write this in prisma

**Comment by nurul3101** [comment, marked as answer]
Hello @f-kb24 👋 
For this case, you would need to use [$queryRaw](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access#queryraw). Prisma has support for [aggregation](https://www.prisma.io/docs/concepts/components/prisma-client/aggregation-grouping-summarizing#aggregate) but that would not help in this use case of adding over two separate columns.
Let me know if you face any issue in implementing this with `$queryRaw`

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
